Makefile

```diff
@@ -512,6 +511,7 @@ prd/embed.html: src/index.mako.html \
 prd/img/: src/img/*
 	mkdir -p $@
 	cp -R $^ $@
+	cp -f $@/favicon.ico $@/../
```

Comment:
might as well copy the ones from src/img as well then. Like this we also have the favicon in dev mode.